src/options/options.requests.ts

```diff
@@ -49,6 +53,7 @@ export interface CalculateGammaResponse {
 
 export interface GreekQueryRequest extends PanopticRequest {
   address: string;
+  panopticPool?: string,
```

Comment:
It looks like you could just have GreekQueryRequest extend PanopticPoolRequest like the others. But maybe this was intentional because we have the possibility of making it so the params here are a token pair instead of a PanopticPool?

Reply:
The motivation for leaving this optional was to allow the possibility of using either the "internal" greek methods (no need for PanopticPool address), or the contract methods (need PanopticPool address). Easy enough to choose just one of the cases and change appropriately.